wordview/mwes/mwe.py

```diff
@@ -95,7 +106,7 @@ def build_counts(self, counts_filename: str = "") -> Union[None, Dict]:  # type:
             None when no counts_filename is provided, otherwise res which is a dictionary of counts.
         """
         logger.info("Creating counts...")
-        res = get_counts(
+        res = self.get_counts(
             df=self.df, text_column=self.text_column, mwe_types=self.mwe_types
```

Comment:
self.get_counts signature has changed and does not accept arguments anymore. Can you please update this and the docstring accordingly?